EasyRPG/TestGame
# Add a Change Tileset testcase and update the Testgames 2000 and 2003

TestGame 2000:
- Add the following EasyRPG RTP assets for use in testcases: CharSet/People2 (for use in the Transitions test to no longer rely on a Don Miguel RTP asset), ChipSet/Dungeon, ChipSet/Exterior, ChipSet/Interior, ChipSet/Ship, ChipSet/World2 (for use in the Change Tileset testcase).
- Update the following EasyRPG RTP assets to match with the ones currently present: CharSet/Chara1 (Actor1), GameOver/GameOver. Also update the asset FaceSet/Chara1 since it uses graphics from the EasyRPG RTP.
- Remove the following unused assets: Title/Title3, Title/Title4. Keeping the assets Title/Title1 since used as a cover for the game on the Android version and keeping the System2 and BattleWeapon folders since used to be used as a way to detect the engine in the past.
- Rename the assets ChipSet/Basis to ChipSet/World and ChipSet/Basis-flat to ChipSet/World-flat.
- Rename some maps in the maptree to avoid having maps with the same name when looking at it.
- Add tests to the Music Playing testcase to play non-existing music, as well as one to check the Midi Play Location and if the Current BGM has completed a loop.
- Add tests to set a variable to Money, Party Size, Save count, Battle count, Win count, Loss count, Escape count, Item is Equipped, Item 0 is Equipped (to check if an empty slot is present) to the Control Variables testcase, as well as a test battle to test to update the win, loss and escape counts.
- Add a test to set a variable to the Timer's Remaining Time in the Timer Operations testcase.
- Add a Change tileset testcase.
- Add a test to change the font used in the Change System testcase, as well as one to test the text there.

TestGame 2003:
- Remove the placeholder ultimate_rt_eb.dll file that was used to detect the game and replace it by an EasyRPG.ini file.
- Fix some typos for the battle lines.
